unit/doc_builder_wrapper/doc_builder_wrapper_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+describe DocBuilderWrapper do
```

Comment:
Actualy I wouldn't call this unit tests
Unit tests is test that perform check if full isolation, check for exact specific methods, usually with Mocks, usually very fast and do not require any deps

Since all those tests require DocBuilder to be installed, I think it's more like an `integration` tests that `unit`

Whay do you think? It's an open question :)

Reply:
I agree. A basic utility is required for these tests to work. 

Let's allocate a separate spec directory for integration tests